src/Rootfs.jl

```diff
@@ -391,14 +391,19 @@ function choose_shards(p::Platform;
 end
 
 """
-    supported_platforms()
+    supported_platforms(;exclude::Vector=[])
 
 Return the list of supported platforms as an array of `Platform`s.  These are the platforms we
 officially support building for, if you see a mapping in `get_shard_hash()` that isn't
 represented here, it's probably because that platform is still considered "in beta".
+
+Platforms can be excluded from the list by specifying an array of platforms to `exclude`.
+i.e. `supported_platforms(exclude=[Windows(:i686),Windows(:x86_64)])`
 """
-function supported_platforms()
-    return [
+function supported_platforms(;exclude::Vector=[])
```

Comment:
Let's have `exclude` be a `Union{Vector{<:Platform},Function}`, then have an inner method definition that does something like:

```
exclude_platforms!(exclude::Function, platforms) = filter(exclude, platforms)
function exclude_platforms!(exclude::Vector{<:Platform}, platforms)
	for e in exclude
		filter!(x -> x != v, platforms)
	end
	return platforms
end
```

Reply:
That will make it easy to do things like turn off all `MacOS` builds, even if an armv7l one appears in the future.  :)

Reply:
Yeah, makes sense. Added